prepare/templates/rag_eval.py

```diff
@@ -0,0 +1,299 @@
+from unitxt import add_to_catalog
+from unitxt.templates import InputOutputTemplateWithCustomTarget, TemplatesList
+
+
+def add_rag_templates(
+    templates_dict, task, out_field="textual_label", reference_field="number_val"
+):
```

Comment:
maybe we should split this file? It's unreadable imo and should be used by other users as well ideally @elronbandel wdyt?

Reply:
did you open an issue for it?